content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,156 @@
+   - Provide descriptions for the pins to aid in testing other designs. 
+
+   <img src="images/info_yaml.png" alt="yamlinfo" width="550"/>
+
+    - Save your changes by clicking the commit button.
+
+    <img src="images/commitinfo.png" alt="commitinfo" width="550"/>
+<!-- 
+![info.yaml](images/info_yaml.png)  -->
+
+<!-- ![Commit the modified info.yaml](images/commitinfo.png) -->
+##  Edit the `info.md` File in the Docs Directory
+
+  **Update Documentation**:
+   - Include detailed instructions on how to experiment with your design. This documentation is useful for users receiving the chip and for you to recall design details later. Note that leaving the info.md unmodified will cause the 'docs' action to fail. Save and commit your changes. Commit can be done in similar way as in Step 2.
+
+   <img src="images/documentation.png" alt="doc" width="550"/>
+<!-- ![Documentation](images/documentation.png) -->
+
+
+## Wokwi Actions
+
+1. **Access Actions Tab**:
+   - Go to the ‘Actions’ tab in your repository to view results. Note that each commit will trigger these actions to run again. Find out more: how the GitHub actions work to build the GDS (link to zero to asic GDS terminology page) files for your project, you can check here (link to /making_asics).
+    
+2. **Actions for Wokwi**:
+
+    - docs: This action generates a preview of how your documentation will look.
+    - fpga:
+    - gds: Action to generate the gds for your project, results for warnings, utilization statistics, and cell usage.
+    - wokwi test:
+    
+
+    <img src="images/actions.png" alt="actionslist" width="300"/>
+
+    Note that red indicates failure and green indicates success. Only the actions 'docs' and 'gds' need to be green
+    Now, the Action results are to be watched to resolve the action failures.
+
+## Troubleshoot Action Failures 
+
+1. **Docs Action**:
+
+   - This action generates a preview of how your documentation will look.
+   - Check the action result to get the status of the check. 
+
+   <img src="images/CheckActions.png" alt="Docs Action Result" width="550"/>
+   
+   - View the Errors and Modify your Docs based on the Error showed, commit the new info.md for actions to rerun. [Tips]() for common cause of errors is shown in the end of this guide.
+
+   <img src="images/checkerrors.png" alt="Docs Action error" width="550"/>
+
+   - Commit will redpo the check and observe the status. 
+
+    <img src="images/changes_docscheck_success.png" alt="Docs Action success" width="550"/>
+
+    - Download the PDF to verify formatting and content.
+
+    <img src="images/actions_pdf.png" alt="Docs Action pdf" width="300"/>
+
+
+<!-- ![Docs Action Result](images/CheckActions.png)
+![Docs Error Result](images/checkerrors.png) -->
+<!--    
+![Docs Success Result](images/changes_docscheck_success.png)
+![Docs Success PDF](images/actions_pdf.png) -->
+
+2. **GDS Action**:
+
+   - This action results in warnings, utilization statistics, and cell usage.
```

Comment:
This action creates the GDS files needed to manufacture your design. You can also see any warnings, utilization statistics and the types of cells used to create your design.